otwcode/otwarchive
# AO3-6659 Option to ban a user and delete all their creations includes collections the user doesn't own

# Pull Request Checklist

* [X ] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [X ] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [X ] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [ X] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [ X] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6659

## Purpose

What does this PR do?

Changed the way user collections are counted. It is using the logic for  sole-owned collections.

## Testing Instructions

How can the Archive's QA team verify that this is working as you intended?

See Jira.

If you have a Jira account with access, please update or comment on the issue
with any new or missing testing instructions instead.

## References

Are there other relevant issues/pull requests/mailing list discussions?

## Credit

Claire C, she/her

*Please note that if you do not fill in this section, we will use your GitHub account name and
they/them pronouns.*
